pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -32,6 +32,7 @@ import cockpit from "cockpit";
 import 'cockpit-components-modifications.css';
 
 const _ = cockpit.gettext;
+tId = undefined;
```

Comment:
this should be `let tId` be part of the component, we don't want to introduce new global variables.